packages/ckeditor5-editor-classic/src/classiceditor.ts

```diff
@@ -85,8 +85,10 @@ export default class ClassicEditor extends ElementApiMixin( Editor ) {
 		this.model.document.createRoot();
 
 		const shouldToolbarGroupWhenFull = !this.config.get( 'toolbar.shouldNotGroupWhenFull' );
+		const useMenuBar = !!this.config.get( 'menuBar' );
```

Comment:
The problem with this is that we're going to use `config.define( 'menuBar', ... )` for the default menu bar configuration soon. This will always be true. Either we don't use `config.define()` or we have to come up with another config.